editor/app/(api)/p/access/[policy]/route.ts

```
@@ -0,0 +1,247 @@
+
+type Context = {
+  params: Promise<Params>;
+};
+
+async function reqformdata(
+  req: NextRequest,
+  contenttype: HeaderContentType
+): Promise<FormData> {
+  switch (contenttype) {
+    case "application/json":
+      const json = await req.json();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)